std/log/logger.ts

```diff
@@ -161,4 +173,8 @@ export class Logger {
   ): T | undefined {
     return this._log(LogLevels.CRITICAL, msg, ...args);
   }
+
+  get _handlers(): BaseHandler[] {
+    return this.#handlers;
+  }
```

Comment:
Why underscore?

Reply:
I had assumed that logger instance handlers didn't need to be public.
It was only being expected by an internal test.
I've pushed a commit that restores it to being public and added in mod_test that users can access and use the handlers member to mutate logger handlers.

(I've kept a `#handlers` member and provided public property getter and setter just for consistency.  Unlike logName and logLevel the handlers list has no particular reason to need to be private and have property accesors)